staging/src/k8s.io/apiserver/pkg/endpoints/installer.go

```diff
@@ -573,6 +574,7 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 				handler = restfulGetResource(getter, exporter, reqScope)
 			}
 			handler = metrics.InstrumentRouteFunc(action.Verb, resource, subresource, handler)
+			handler = genericfilters.RestfulWithCompression(handler, a.group.Context)
```

Comment:
> where should the escape flag be read from? does this involve adding a configuration variable & a command line flag to apiserver options?

Open an issue and fix it in a separate pull.

@liggitt I don't think this is a setting we're likely to keep long term.  If it works through a release, I think we make this unconditional.  Please keep that in mind when choosing between flag and env var.